src/core/primitives/Dropdown/Select/index.tsx

```
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+type SelectPrimitiveProps = {  
+  children?: React.ReactNode, 
+  options: string[],
+  value?: string | null,
+  onChange?: (value: string) => void;
+}
+
+const SelectPrimitive = ({ children, options, value, onChange, ...props }:SelectPrimitiveProps) => {
+
+    const [isOpen, setIsOpen] = useState(false);
```

ESLint

[error] Block must not be padded by blank lines.

(padded-blocks)